scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -327,10 +350,26 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
                 <Nav.SideBarTabs
                     activeTab={this.state.sideWidget?.id}
                     onActiveTabChange={(val) => {
+                        if (val == null && closedThroughAction.closed == false) {
+                            closedThroughAction.closed = true;
+                        }
+
+                        const widgetFromPreferences = sdApi.preferences.get(PINNED_WIDGET_USER_PREFERENCE_SETTINGS);
+
+                        const isWidgetPinned = (() => {
+                            if (widgetFromPreferences._id == val) {
```

Comment:
shouldn't we only take preferences into account to come up with a initial value in the constructor?